docs/en_US/NAS/NasGuide.md

```diff
@@ -0,0 +1,311 @@
+# Guide: Using NAS on NNI
+
+```eval_rst
+.. contents::
+
+.. Note:: The APIs are in an experimental stage. The current programing interface is subject to change.
+```
+
+![](../../img/nas_abstract_illustration.png)
+
+Modern Neural Architecture Search (NAS) methods usually incorporate [three dimensions][1]: search space, search strategy, and performance estimation strategy. Search space often contains a limited neural network architectures to explore, while search strategy samples architectures from search space, gets estimations of their performance, and evolves itself. Ideally, search strategy should find the best architecture in the search space and report it to users. After users obtain such "best architecture", many methods use a "retrain step", which trains the network with the same pipeline as any traditional model.
+
+## Implement a Search Space
+
+Assuming now we've got a baseline model, what should we do to be empowered with NAS? Take [MNIST on PyTorch](https://github.com/pytorch/examples/blob/master/mnist/main.py) as an example, the code might look like this:
+
+```python
```

Comment:
after implementing codes for search space, do we need to talk about the SearchSpace json file?